### PR TITLE
Render

### PR DESCRIPTION
**Pull request by axis7818**

### CODE REVIEW

**Review by martynasbudvytis — APPROVED** [review]
:shipit:

File: src/voyager-core/include/Application.h [hunk 1]
```diff
@@ -24,9 +26,17 @@ class Application : public EventCallbacks {
 
    // getters/setters
    ApplicationType getType() { return this->type; }
+
+   std::string getResourceDir() { return this->resource_dir; }
+   void setResourceDir(std::string resource_dir) {
```
Comment (andrewjpiro) [inline src/voyager-core/include/Application.h:31]: This setter is the only one on multiple lines. We should make them consistent

File: src/voyager-core/include/Application.h [hunk 2]
```diff
@@ -45,7 +55,9 @@ class Application : public EventCallbacks {
 
    std::vector< std::shared_ptr<Entity> > things;
 
-   LoopTimer timer = LoopTimer(10); // 10ms max time step
+   LoopTimer timer = LoopTimer(10); // 10ms ma time step
```
Comment (andrewjpiro) [inline src/voyager-core/include/Application.h:58]: 'ma' typo in comment

File: src/voyager-render/include/Camera.h [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#pragma once
+#ifndef _CAMERA_H
+#define _CAMERA_H
+
+#include <memory>
+
+#include <glm/glm/gtc/type_ptr.hpp>
+
```
Comment (andrewjpiro) [inline src/voyager-render/include/Camera.h:8]: So much space. Do we want this much?

File: src/voyager-render/include/RenderEngine.h [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+   std::shared_ptr<Camera> getCamera() { return this->camera; }
+   void setCamera(std::shared_ptr<Camera> camera) { this->camera = camera; }
+
+   std::shared_ptr<Program> getProgram() { return this->program; }
+   void setProgram(std::shared_ptr<Program> program) {
+      this->program = program;
+   }
+
+   std::shared_ptr<WindowManager> getWindow() { return this->window; }
+   void setWindow(std::shared_ptr<WindowManager> window) {
```
Comment (andrewjpiro) [inline src/voyager-render/include/RenderEngine.h:34]: There are more instances of multiple line setters here

**Review by andrewjpiro — COMMENTED** [review]
Looks good. Made some comments on style

**Pull request merged**